README.md

```diff
@@ -44,11 +44,94 @@ The object has the following schema (validated [here](./lib/index.js) using [Joi
   * `done: (err: Error, key: String)` - A function that takes an error as the first parameter and the bucket key as the second parameter.
 
 **Optional**
+* `enabled: Boolean default(true)` - if `true` rate limiting will be enabled you can set it to `false` on route settings to disable rate limiting
```

Comment:
what if we call it `allRoutes` and it can only be set at the global plug in level not at the route level? if a route has the plugin defined, it wants to enable it so no need for the flag at the route level